docs/version0.27.md

```diff
@@ -55,4 +56,9 @@ For more information about this type of operation, see [GC copy forward operatio
 
 You can revert to the behavior in earlier releases by setting [`-Xgc:breadthFirstScanOrdering`](xgc.md#breadthfirstscanordering) when you start your application.
 
+### Stop parsing the JAVA_OPTIONS environment variable
+
+The 0.24 release starting parsing the JAVA_OPTIONS environment variable. This was added in error and has been removed. The [_JAVA_OPTIONS environment variable](cmdline_specifying.md)
```

Comment:
- Should be "started" instead of "starting".

- Can we have "This variable" instead of just "This", which is ambiguous.

- I think we should mention that the new option behaves differently otherwise it looks like all we've done is add an underscore to the variable name. eg 

  > The _JAVA_OPTIONS environment variable (with different behavior) is added for compatibility.

Reply:
Updated as suggested.